src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/tools/ansyn.annotations.visualizer.ts

```diff
@@ -280,11 +274,6 @@ export class AnsynAnnotationsVisualizer extends BaseImageryPlugin {
 	onInit() {
 		super.onInit();
 		this.annotationsVisualizer = this.communicator.getPlugin(AnnotationsVisualizer);
-		this.store$.select(selectAnnotationMode)
```

Comment:
why?

Reply:
this is causing a bug when moving from cesium to ol! you will be in drawing mode even if you're not! and we already doing it in line 97  the annoatationModeChange$ observable (basically it's likedoing it twise)